pip-install-req.txt

```diff
@@ -10,7 +10,7 @@ git+https://github.com/emory-libraries/pidman-client.git@release-1.1.2#egg=pidse
 python-magic
 git+https://github.com/eliben/pycparser@release_v2.14
 pycrypto
-celery
```

Comment:
Would be helpful to include the reasons for using a specific celery reason. Could include in the DEVNOTES or the CHANGELOG.